src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java

```diff
@@ -14,4 +21,20 @@
 public class WalletController {
 
     private final WalletService walletService;
+
+    @GetMapping(value = "/wallets")
+    ResponseEntity<List<WalletDTO>> getWallets() throws AppException {
+        try {
+            List<WalletDTO> allWallets = walletService.getAllWallets();
+
+            if (allWallets.isEmpty()) {
+                return ResponseEntity.status(HttpStatus.NO_CONTENT).body(allWallets);
+            }
+
+            return ResponseEntity.status(HttpStatus.OK).body(allWallets);
+
+        } catch (Exception e) {
```

Comment:
Po co ten try catch? Nie powinno być to zrobione w serwisie? Jak już ten try catch to poleciał error i co dalej, pasuje jakiś log dodać

Reply:
Done, try/catch section has been moved to service layer. Proper log has been added.